boards/calliope-mini/include/periph_conf.h

```diff
@@ -78,9 +71,9 @@ static const timer_conf_t timer_config[] = {
  * @{
  */
 #define RTT_NUMOF           (1U)
-#define RTT_DEV             (1)             /* NRF_RTC1 */
-#define RTT_MAX_VALUE       (0x00ffffff)
-#define RTT_FREQUENCY       (1024)
+#define RTT_DEV             (1U)             /* NRF_RTC1 */
+#define RTT_MAX_VALUE       (0x00ffffffU)
```

Comment:
not sure, but IMHO all hex numbers are implicitly `unsigned`, so no need for the `U` in this case.